package.json

```diff
@@ -111,9 +113,11 @@
     "sri-validate": "node build/sri-update --validate",
     "fmt": "prettier --write .",
     "prepare": "husky",
-    "prebuild": "node ./build/check-node-version.js",
+    "prebuild": "node ./build/check-node-version.js && npm run unpatch",
+    "predevelop": "npm run unpatch",
     "pretest": "node ./build/check-node-version.js",
-    "postbuild": "prettier --write ./locales/_template.json ./doc/rule-descriptions.md"
+    "postbuild": "prettier --write ./locales/_template.json ./doc/rule-descriptions.md && npm run patch",
```

Comment:
I suspect you mean to unpatch here. I'm not sure about this unpatching at the end of the flow. It feels like there are potential gaps there, especially with develop. Might be better if we had one script that unpatches, copies, then patches, which runs before develop and build. That feels like a slightly safer way to ensure the files in `patch/unpatched` are always up to date.